client/layout/masterbar/logged-in.jsx

```diff
@@ -653,112 +764,64 @@ class MasterbarLoggedIn extends Component {
 		);
 	}
 
+	renderBackHomeButton() {
+		const { translate } = this.props;
+
+		return (
+			<Item
+				className="masterbar__item-back"
+				icon="chevron-left"
+				tooltip={ translate( 'Back' ) }
+				url={ this.getHomeUrl() }
+			/>
+		);
+	}
+
 	render() {
 		const { isInEditor, isCheckout, isCheckoutPending, isCheckoutFailed, loadHelpCenterIcon } =
 			this.props;
-		const { isMobile } = this.state;
+		const { isMobile, isResponsiveMenu } = this.state;
 
+		// Checkout flow uses it's own version of the masterbar
 		if ( isCheckout || isCheckoutPending || isCheckoutFailed ) {
 			return this.renderCheckout();
 		}
 
-		if ( this.props.isMobileGlobalNavVisible ) {
-			return (
-				<>
-					<Masterbar className="masterbar__global-nav">
-						<div className="masterbar__section masterbar__section--left">
-							{ this.renderSidebarMobileMenu() }
-							{ this.renderGlobalMySites() }
-							{ this.renderReader() }
-						</div>
-						<div className="masterbar__section masterbar__section--right">
-							{ this.renderSearch() }
-							{ this.renderCart() }
-							{ this.renderMe() }
-							{ loadHelpCenterIcon && this.renderHelpCenter() }
-							{ this.renderNotifications() }
-						</div>
-					</Masterbar>
-				</>
-			);
-		}
-
-		if ( this.props.isUnifiedSiteView ) {
+		// Editor specific masterbar, only shows back to home button and help center as these are hidden on mobile views
+		// from the desktop version of the editor.
+		// The desktop version of the editor has no masterbar at all so we only do this for mobile.
+		if ( isInEditor && ( isMobile || isResponsiveMenu ) ) {
 			return (
-				<Masterbar className="masterbar__unified">
```

Comment:
We can now remove the entire `.masterbar__unified` selectors from the CSS file.

Reply:
Global nav too I guess: https://github.com/Automattic/wp-calypso/pull/92472/commits/7c98adc355d1c303bfdcf5e41fd59912c1b8cf47